Misc/NEWS.d/next/Library/2019-05-01-20-41-53.bpo-36772.fV2K0F.rst

```diff
@@ -0,0 +1,2 @@
+functools.lru_cache() can now be used as a straight decorator rather than as
```

Comment:
I think this should say "in addition to" instead of "rather than as".

Reply:
Will do.